src/VisualStudio/Core/Def/Implementation/EditAndContinue/VisualStudioActiveStatementProvider.cs

```diff
@@ -37,7 +37,7 @@ public Task<ImmutableArray<ActiveStatementDebugInfo>> GetActiveStatementsAsync(C
 
                 // No exception should be thrown in case of errors on the debugger side. 
                 // The debugger is responsible to provide telemetry for error cases.
-                var workList = DkmWorkList.Create(null); 
+                var workList = DkmWorkList.Create(_ => { completion.TrySetResult(ImmutableArray<ActiveStatementDebugInfo>.Empty); });
```

Comment:
>var workList = Dk [](start = 16, length = 17)

Lets add a comment explaining that the callback should not be called, but it's there to guarantee that the task completes and a hang is avoided.